cg/constants/housekeeper_tags.py

```diff
@@ -202,18 +202,22 @@ class BalsamicProtectedTags:
         ["gisaid-log"],
         ["gisaid-csv"],
     ],
+    Workflow.RAREDISEASE: [
+        [HermesFileTag.LONG_TERM_STORAGE],
+    ],
     Workflow.RNAFUSION: [
-        [AnalysisTag.FUSION, AnalysisTag.ARRIBA],
-        [AnalysisTag.FUSION, AnalysisTag.STARFUSION],
-        [AnalysisTag.FUSION, AnalysisTag.FUSIONCATCHER],
-        [AnalysisTag.FUSIONINSPECTOR],
-        [AnalysisTag.FUSIONREPORT, AnalysisTag.RESEARCH],
-        [AnalysisTag.FUSIONINSPECTOR_HTML, AnalysisTag.RESEARCH],
-        [AnalysisTag.ARRIBA_VISUALIZATION, AnalysisTag.RESEARCH],
-        [AnalysisTag.MULTIQC_HTML, AnalysisTag.RNA],
-        [HK_DELIVERY_REPORT_TAG],
-        [AnalysisTag.VCF_FUSION],
-        [AnalysisTag.GENE_COUNTS],
+        [HermesFileTag.LONG_TERM_STORAGE],
+        [AnalysisTag.FUSION, AnalysisTag.ARRIBA],  # legacy
+        [AnalysisTag.FUSION, AnalysisTag.STARFUSION],  # legacy
+        [AnalysisTag.FUSION, AnalysisTag.FUSIONCATCHER],  # legacy
+        [AnalysisTag.FUSIONINSPECTOR],  # legacy
+        [AnalysisTag.FUSIONREPORT, AnalysisTag.RESEARCH],  # legacy
+        [AnalysisTag.FUSIONINSPECTOR_HTML, AnalysisTag.RESEARCH],  # legacy
+        [AnalysisTag.ARRIBA_VISUALIZATION, AnalysisTag.RESEARCH],  # legacy
+        [AnalysisTag.MULTIQC_HTML, AnalysisTag.RNA],  # legacy
+        [HK_DELIVERY_REPORT_TAG],  # legacy
+        [AnalysisTag.VCF_FUSION],  # legacy
+        [AnalysisTag.GENE_COUNTS],  # legacy
```

Comment:
why do we need to keep the legacy ones? would they otherwise be removed?

Reply:
Yes, exactly. Ideally, we would extract all the files with those tags and update HK include the `long-term-storage` tag but this is an easier approach 😓 . Hopefully, someone will do it when we start deprecating MIP, which has a lot of these legacy tags